elastic/kibana
# [Synthetics] alert config read only

## Summary

Relates to https://github.com/elastic/kibana/issues/150417
Resolves https://github.com/elastic/kibana/issues/150472

Adds alert config to ICMP monitors.

Also allows alert config to be specified for project monitors.

<img width="1221" alt="Screen Shot 2023-02-07 at 2 10 36 PM" src="https://user-images.githubusercontent.com/11356435/217363925-1dd1188b-ad7c-4cb8-b5b3-0d7464f875d7.png">
<img width="1257" alt="Screen Shot 2023-02-07 at 4 04 30 PM" src="https://user-images.githubusercontent.com/11356435/217365266-6aca2b82-da3d-445d-8dfc-9dabd1df07c6.png">

## Testing
1. Navigate to Add/Monitor and click on ICMP monitor
2. Ensure the alert toggle appears and is able to be interacted with
3. Testing for project monitor alert config will have to be done after https://github.com/elastic/kibana/issues/150417, but api integration tests have been added.
